client/app/components/composites/Avatar/Avatar.js

```diff
@@ -0,0 +1,71 @@
+import { Component, PropTypes } from 'react';
+import r, { div, img } from 'r-dom';
+import _ from 'lodash';
+import classNames from 'classnames';
+import { className } from '../../../utils/PropTypes';
+import ArrowDropdown from './ArrowDropdown';
+
+import css from './Avatar.css';
+
+class Avatar extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.handleHover = this.handleHover.bind(this);
+
+    this.state = {
+      menuOpen: false,
+    };
+
+    this.menuCloseDelay = 300;
+  }
+
+  handleHover(event) {
+    if (event && event.type === 'mouseover') {
+      if (this.debouncedClose) {
+        this.debouncedClose.cancel();
+        this.debouncedClose = null;
+      }
+      this.setState({ menuOpen: true }); // eslint-disable-line react/no-set-state
+    } else if (event && event.type === 'mouseout') {
+      if (!this.debouncedClose) {
+        this.debouncedClose = _.debounce(() => {
+          this.setState({ menuOpen: false }); // eslint-disable-line react/no-set-state
+        }, this.menuCloseDelay);
+        this.debouncedClose();
+      }
+    }
+  }
```

Comment:
Using JavaScript to handle hover state is bound to leave the menu open in weird situations, especially on mobile devices.

Maybe not version 1, but hopefully we could spend some time getting these dropdowns open with purely CSS if that's possible.